spec/services/preserved_object_handler_update_version_spec.rb

```diff
@@ -101,30 +116,41 @@
         let(:updated_po_db_timestamp_msg) { "#{exp_msg_prefix} PreservedObject updated db timestamp only" }
         let(:updated_pc_db_timestamp_msg) { "#{exp_msg_prefix} PreservedCopy updated db timestamp only" }
 
-        it "entry version stays the same" do
-          pocv = po.current_version
+        it "PreservedCopy version stays the same" do
           pcv = pc.version
           po_handler.update_version
-          expect(po.reload.current_version).to eq pocv
           expect(pc.reload.version).to eq pcv
         end
-        it "entry size stays the same" do
+        it "PreservedObject current_version stays the same" do
+          pocv = po.current_version
+          po_handler.update_version
+          expect(po.reload.current_version).to eq pocv
+        end
+        it "PreservedCopy size stays the same" do
           expect(pc.size).to eq 1
           po_handler.update_version
           expect(pc.reload.size).to eq 1
         end
-        it 'updates status of PreservedCopy to "ok"' do
-          skip("should it update status of PreservedCopy?")
-          expect(pc.status).to eq Status.unexpected_version
+        it 'does not update PreservedCopy last_audited field' do
+          orig_timestamp = pc.last_audited
           po_handler.update_version
-          expect(pc.reload.status).to eq Status.ok
```

Comment:
seems like it maybe it should update to `Status.unexpected_version`?  but i'm not sure about that, and also not sure if i'm forgetting whether we already agreed on what the behavior should be in story time (or whether this really is still an open question.

i do see a TODO in the POH app code about whether timestamp/status should get updated in the case of unexpected version, so wouldn't expect a test for that to pass anyway.  but might be worth leaving in the skipped test?  though i guess a test would naturally get added if/when that TODO gets addressed.

https://github.com/sul-dlss/preservation_catalog/blob/master/app/services/preserved_object_handler.rb#L176-L181

Reply:
Based on what is in #287, this was already changed to:

```
        it 'does not update status of PreservedCopy' do
          orig_status = pc.status
          po_handler.update_version
          expect(pc.reload.status).to eq orig_status
        end
```